awx/main/migrations/_create_system_jobs.py

```diff
@@ -15,9 +16,41 @@
 only if new system job templates were created (i.e. new database).
 '''
 
+def create_collection_jt(apps, schema_editor):	
+    SystemJobTemplate = apps.get_model('main', 'SystemJobTemplate')	
+    Schedule = apps.get_model('main', 'Schedule')	
+    ContentType = apps.get_model('contenttypes', 'ContentType')	
+    sjt_ct = ContentType.objects.get_for_model(SystemJobTemplate)	
+    now_dt = now()	
+    random_time = now_dt + timedelta(minutes=random.randint(-30,30))	
+    random_schedule_time = random_time.strftime('%Y%m%dT%H%M%SZ')
+    insights_enabled = getattr(settings, 'INSIGHTS_TRACKING_STATE', False)
```

Comment:
Now that enablement is controlled by a setting, is there any code necessary to prevent `INSIGHTS_TRACKING_STATE` from showing up in the UI?  Would it make more sense to just remove this setting *entirely* now that it doesn't actually affect anything?